gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Mon Sep 19 08:21:35 CEST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
Do not hide the gradle updates in another feature branch.

Reply:
We need to add a reference to the Trello ticket and need to copy the title and description of the ticket linked to the feature developed into the description of the pull request.